.github/workflows/build.yml

```diff
@@ -320,13 +314,11 @@ jobs:
      - name: Unpack jdk
        run: |
          unzip "${HOME}/${{matrix.version}}-${{matrix.os}}-${{matrix.variant}}/OpenJDK.zip" -d "${HOME}/${{matrix.version}}-${{matrix.os}}-${{matrix.variant}}"
-
      - name: Set root of jdk image dir
        shell: pwsh
        run: |
          $imageroot=$(find "${HOME}/${{matrix.version}}-${{matrix.os}}-${{matrix.variant}}" -name release -type f)
          echo "TEST_JDK_HOME=$(dirname "${imageroot}")" | Out-File -FilePath "$env:GITHUB_ENV" -Encoding utf8 -Append
-
```

Comment:
Probably worth making this while file identical to current master, ie.remove from this PR